projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader.component.ts

```diff
@@ -61,8 +63,7 @@ export class NgxSkeletonLoaderComponent implements OnInit, AfterViewInit, OnDest
       }
       this.count = 1;
     }
-
-    this.items.length = this.count;
+    this.items.length = this.count || 1;
```

Comment:
What about using the nullish coalescing operator here?
```diff
- this.item.length = this.count || 1;
+ this.item.length = this.count ?? 1;
```

I think it is useable since TypeScript v3.7 and this project is in 3.8.3:
![image](https://user-images.githubusercontent.com/32441291/120390211-b9052980-c32d-11eb-9af6-4cf226009482.png)

Reply:
Won't it be a breaking change for someone using `[count]="0"`, for example?

Reply:
The nullish coalescing operator specically checks for the value not to be NULL or UNDEFINED. So the `||` would be a breaking change, but not the `??`